implementations/contracts/ERC725/ERC725AccountCore.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+// modules
+import "./ERC725XCore.sol";
+import "./ERC725YCore.sol";
+import "../IERC1271.sol";
+
+// libraries
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "../helpers/UtilsLib.sol";
+import "../Utils/ERC725Utils.sol";
+
+// interfaces
+import "../Interfaces/ILSP1_UniversalReceiver.sol";
+import "../Interfaces/ILSP1_UniversalReceiverDelegate.sol";
+
+/**
+ * @title Abstract (Core) implementation of ERC725Account
+ * @dev Bundles ERC725X and ERC725Y, and ERC1271 and allows receiving native tokens.
+ *
+ *  @author Fabian Vogelsteller <fabian@lukso.network>, Jean Cavallera (CJ42), Yamen Merhi (YamenMerhi)
+ */
+
+abstract contract ERC725AccountCore is ERC725XCore, ERC725YCore, ILSP1, IERC1271 {
+    using ERC725Utils for ERC725YCore;
+
+    bytes4 internal constant _INTERFACE_ID_ERC725Account = 0x63cb749b;
+    bytes4 internal constant _INTERFACE_ID_ERC1271 = 0x1626ba7e;
+    bytes4 internal constant _ERC1271FAILVALUE = 0xffffffff;
+    bytes4 internal constant _INTERFACE_ID_LSP1 = 0x6bb56a14;
```

Comment:
those should be calculated `type(ILSP6).interfaceId`
same for the others where we have interfaces